### ISSUE TITLE
Can't run or find "@cypress/react init"..

### ISSUE THREAD

**Issue by pbadenski** [cypress-io/cypress#15450]
Documentation mentions `@cypress/react init`. Does that still exist? I installed @cypress/react as a local package using yarn and it doesn't work. I can't find any executables in `node_modules/.bin` and `grep -r init node_modules/@cypress/react/` doesn't seem to show anything relevant.

**Comment by joshpzero** [comment]
I noticed this too. Also the links to the Examples on component testing didn't seem to work after merging the package into @cypress/react

**Comment by JessicaSachs** [comment]
I must've missed that `init` command... I don't think I've ever used it. It might've gotten dropped in the repo migration. We've been moving around a lot of ops (readme locations, docs, etc). Will look into it and update the documentation with the right links. Thank you!

**Comment by elevatebart** [comment]
Thank you both for informing us of this oversight. 

In the new version of `@cypress/react` we were hoping to use the new [create-cypress-tests](https://www.npmjs.com/package/create-cypress-tests) package to deal with this in a wizard. We should really have updated the docs.

Here is how you should be getting to the same result as using `@react/init`:

```bash
npx create-cypress-tests --component-testing
```

Let me re-test this command in a repo to confirm.

**Comment by elevatebart** [comment]
Sorry guys, 

I just tested this and though it should work, we failed the delivery process of the said wizard.
Right now, the delivered version of `create-cypress-tests` contains no built files.
It is empty...

Woops.

Will fix it ASAP.

**Comment by joshpzero** [comment]
Thanks for the update. I really like this feature. Here's some things I've noted.

I used:
 `npx create-react-app cra-cypress --template typescript`
`npx create-cypress-tests --component-tests`

Problems

- `import '@cypress/react/support';` was missing from support/index.js


Suggestions

- Add script for cy:open and cy:run
- Add example component test for app.js on cy:open execution
- option to remove react-testing-library potentially

**Comment by lmiller1990** [comment]
I don't think having `init` makes much sense now we have a unified runner for all frameworks. The `create-cypress` wizard will take this place. It's still a bit buggy as discussed above, so I collected the suggestions and a number of other bugs I found [into this issue](https://github.com/cypress-io/cypress/issues/15933) where I'll track progress.

`@cypress/react init` is now removed from the docs to avoid any confusion.
